packages/cloud-cognitive/src/components/UserProfileImage/UserProfileImage.js

```diff
@@ -141,24 +135,32 @@ UserProfileImage.propTypes = {
    * Provide an optional class to be applied to the containing node.
    */
   className: PropTypes.string,
-  /**
-   * When passing the icon prop, use either "user" or "group". The values match up to the Carbon Library icons.
-   */
-  icon: PropTypes.oneOf(['user', 'group']),
   /**
    * When passing the image prop, supply a full path to the image to be displayed.
    */
   image: PropTypes.string,
+  /**
+   * When passing the image prop use the imageDecsription prop to describe the image for screen reader.
+   */
+  imageDescription: PropTypes.string,
```

Comment:
This should be required.

Reply:
@sydrosa is right, `imageDescription` needs to be marked as `required`.

Reply:
This was addressed in @qbi11y latest commit that adds custom prop validation for `imageDescription`